### PR TITLE
Bump NWJS to v0.77.0

### PR DESCRIPTION
**Pull request by haslinghuis**
The ARM64 architecture is supported on macOS starting from this release.

https://nwjs.io/blog/v0.77.0/

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Comment by HThuren** [comment]
@haslinghuis @chmelevskij @nerdCopter 
We have an issue with nwjs version 0.77.0 who seems to contain a virus (Trojan:Win32/Wacatac.H!ml) and strange reference to Anolis. At my pc, windows defender catch the virus. Same issue with the files build ind this PR
Version 0.72.0 don't have the issue
Version 0.78.1 works, nice suggestion from @ASDosjani, but have an issue with icon at taskbar

**Pull request merged**
